mainnet/config.yaml

```diff
@@ -1,37 +1,34 @@
-# this sets the logger level 
 environment: production
+# Aethos ServiceManager Proxy Contract
+service_manager_contract: 0x07E26BF8060e33fA3771d88128b75493750515c1
+# Eigenlayer AVS Directory Contract (Ethereum Mainnet)
+eigen_avs_directory_contract: 0x135DDa560e946695d6f155dACaFC6f1F25C1F5AF
+# Internal GRPC server the public server forwards messages to
+internal_task_server_endpoint: 0.0.0.0:9010
+# Middleware DNS and port
+middleware_server_endpoint: task.aethos.network:50051
+# Encompasses operator metrics including eigen metrics
+metrics_server_endpoint: 0.0.0.0:9090
+# Eigen avs node spec compliance https://eigen.nethermind.io/docs/spec/intro
+eigen_avs_node_api_server_endpoint: 0.0.0.0:8080
+# db path
+db_path: /app/data/
+# enable log colors
+enable_log_color: true
 
-# ethereum node urls
-eth_rpc_url: <eth_rpc_url>
-
-# avs service manager proxy address 
-avs_service_manager_address: 0x07E26BF8060e33fA3771d88128b75493750515c1
-# avs directory address (eigenlayer ethereum mainnet)
-avs_directory_address: 0x135DDa560e946695d6f155dACaFC6f1F25C1F5AF
+###############################################################################
+####### TODO: Operators please update below values for your node ##############
+###############################################################################
 
-# registration private key store path and password
+# GRPC server on which the operator listens for messages from the middleware
+public_task_server_endpoint: <your_public_ip_address>:9010
+# Registration private key store path
 registration_private_key_store_path: <path_to_private_key_store>
+# Password to access the private key store
 registration_private_key_password: <password>
-
-# aethos signing private key store path and password
+# Aethos signing private key store path
 aethos_signing_private_key_store_path: <path_to_private_key_store>
+# Password to access the signing private key store
 aethos_signing_private_key_password: <password>
-
-# avs node spec compliance https://eigen.nethermind.io/docs/spec/intro
-node_eigen_api_server_host_and_port: 0.0.0.0:8080
-eigen_metrics_ip_port_address: 0.0.0.0:9090
-
-# grpc server for communication in between operator and aggregator
-node_task_server_host_and_port: 0.0.0.0:9010
-
-# grpc server on which the operator receives tasks
-node_task_server_host_and_port_to_broadcast: <your_public_ip_address>:9010
-
-# address which the aggregator listens on for operator signed messages
-aggregator_server_ip_port_address: task.aethos.network:50051
-
-# db path
-db_path: /app/data/
-
-# enable log colors
-enable_log_color: true
\ No newline at end of file
+# Comma separated list of Mainnet archive node URLs
+eth_rpc_urls: <https://rpc.abc.com,https://rpc.def.com>
```

Comment:
Here they should do an array [] so maybe just give an array with these example values